platform/src/test/java/org/hillview/test/FreqKTest.java

```diff
@@ -39,49 +39,56 @@
 import static org.junit.Assert.assertTrue;
 
 public class FreqKTest extends BaseTest {
-    private void filterTest(@Nullable FreqKList fkList) {
+
+    public void checkList(FreqKList fkList, Schema schema) {
```

Comment:
Can you also check that counts are non-negative?
We've seen some of these in the past.